### PR TITLE
Added day1 solution and question

### PR DESCRIPTION
**Pull request by tejaswinni51**
issue #2

### CODE REVIEW

File: day1/README.md [hunk 1]
```diff
@@ -82,6 +82,9 @@ array.forEach(function(currentValue, index, arr), thisValue)
 ## Practise questions
 
 To be added...
```
Comment (MadhavBahl) [inline day1/README.md]:
```suggestion
[Feel free to add more and open a Pull Request]

```

**Comment by MadhavBahl** [comment]
Hi @tejaswini22199 
Looks good,

Added a small comment

**Comment by tejaswinni51** [comment]
Sir could you please let me know if  there is anything to change?

**Comment by MadhavBahl** [comment]
Hi @tejaswini22199 
This looks good,
Thanks for the contribution 😊

**Pull request merged**